packages/analytics/src/integrations/Omnitracking/definitions.ts

```diff
@@ -460,6 +461,23 @@ export const trackEventsMapper: Readonly<OmnitrackingTrackEventsMapper> = {
     tid: 2923,
     checkoutStep: data.properties.step,
   }),
+  [eventTypes.PAYMENT_INFO_ADDED]: data => ({
+    tid: 2912,
+    checkoutStep: data.properties?.step,
```

Comment:
Here the implementation could use the `getCommonCheckoutStepTrackingData` function as well, I believe.

Reply:
True, also moved the mapping of `selectedPaymentMethod` to the util since it was repeated across 3 event mappings that were also using the `getCommonCheckoutStepTrackingData `. thanks!